modules/jaeger-config-sampling.adoc

```diff
@@ -65,11 +62,36 @@ kind: Jaeger
 metadata:
   name: with-sampling
 spec:
-  strategy: allInOne
   sampling:
     options:
       default_strategy:
         type: probabilistic
-        param: 50
+        param: 0.5
+      service_strategies:
+        - service: alpha
+          type: probabilistic
+          param: 0.8
+          operation_strategies:
+            - operation: op1
+              type: probabilistic
+              param: 0.2
+            - operation: op2
+              type: probabilistic
+              param: 0.4
+        - service: beta
+          type: ratelimiting
+          param: 5
+----
+
+If there are no user-supplied configurations, Jaeger uses the following settings.
 
+.default sampling
```

Comment:
we may remove this `default sampling`. if you add this to the table: `Jaeger sampling options`